### DISCUSSION TITLE
My Tailwind class in my custom page is not work

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#11868] [General]
Hi I use filament and use custom page, in that page some tailwind class is not work like grid, background, text-color etc ...
how to make all tailwind class can be effect in that page 

I also install this theme in my project and make all the step that describe in documentation but noting work 

[https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)

how to fix this pls

**Comment by leandrocfe** [comment]
_If you want to apply Tailwind CSS classes that are not already used in Blade files, you should update your content configuration in tailwind.config.js to also scan for classes inside your directory: './app/Filament/**/*.php'_

**Comment by manishshah-dev** [comment]
@ayemin-tun did you find solution to make tailwind classes work?

**Comment by Saad5400** [comment]
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

**Comment by Uzair76** [comment]
**Just follow these steps for custum page theme and Tailwing not working issue**  
Step 1: run this command in terminal-> "php artisan make:filament-theme" or if you want to specify pannel incase of multiple pannels run "php artisan make:filament-theme admin"
Step 2: flow these three simple steps 

- First, add a new item to the `input` array of `vite.config.js`: `resources/css/filament/admin/theme.css`
-  Next, register the theme in the admin panel provider using `->viteTheme('resources/css/filament/admin/theme.css')`
-  Finally, run `npm run build` to compile the theme

Step 3 : hit "npm run build "

it will configure custum them and tailwind will work fine 
or 
follow the link https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme
